File: ClickServerWeb/WebContent/com/pku/ss/egl/prog/PlayVideo.js
define([],function(){
	egl.defineClass('com.pku.ss.egl.prog', "PlayVideo", ProgramBase, {
		"eze$$fileName" : "com/pku/ss/egl/prog/PlayVideo.egl",
			,
			"eze$$setEmpty": function() {
			}
			,
			"eze$$setInitial": function() {
				this.eze$$setEmpty();
			}
			,
			"eze$$clone": function() {
				var ezert$$1 = this;
				var ezert$$2 = new egl.com.pku.ss.egl.prog.PlayVideo();
				return ezert$$2;
			}
			,
			"eze$$getAnnotations": function() {
				if(this.annotations === undefined){
					this.annotations = {};
					this.annotations["XMLRootElement"] = new egl.eglx.xml.binding.annotation.XMLRootElement("PlayVideo", null, false);
				}
				return this.annotations;
			}
			,
			"eze$$getFieldInfos": function() {
				if(this.fieldInfos === undefined){
					var annotations;
					this.fieldInfos = new Array();
				}
				return this.fieldInfos;
			}
			,
			"main": function() {
				egl.eglx.lang.SysLib.startCmd("video\\Rolling In The Deep.MP4");
			}
			,
			"toString": function() {
				return "[PlayVideo]";
			}
		}
	);
});

File: ClickServerWeb/src/com/pku/ss/egl/prog/PlayVideo.java
package com.pku.ss.egl.prog;
import org.eclipse.edt.javart.resources.*;
import org.eclipse.edt.javart.*;
import eglx.lang.SysLib;
import org.eclipse.edt.runtime.java.eglx.lang.EAny;
import java.lang.Object;
@SuppressWarnings("unused")
@javax.xml.bind.annotation.XmlRootElement(name="PlayVideo")
public class PlayVideo extends ProgramBase {
	private static final long serialVersionUID = 10L;
	
	public static void main(String... ezeargs) throws Exception {
		StartupInfo info = new StartupInfo( "PlayVideo", "com/pku/ss/egl/prog/PlayVideo.properties", ezeargs );
		RunUnit ru = new JSERunUnit( info );
		org.eclipse.edt.javart.Runtime.setStaticRunUnit( ru );
		ru.start( new PlayVideo() );
		ru.exit();
	}
	public PlayVideo() {
		super();
	}
	{
		if(org.eclipse.edt.javart.Runtime.getRunUnit().getActiveExecutable() == null)
		org.eclipse.edt.javart.Runtime.getRunUnit().setActiveExecutable(this);
		ezeInitialize();
	}
	public void ezeInitialize() {
	}
	public void main() {
		SysLib.startCmd("video\\Rolling In The Deep.MP4");
	}
}
